imagersite/imager_images/tests.py

```diff
@@ -0,0 +1,130 @@
+from __future__ import unicode_literals
+from shutil import rmtree
+
+from django.contrib.auth.models import User
+from django.conf import settings
+from django.test import TestCase
+import factory
+from faker import Faker
+
+from .models import Album, Photo
+
+fake = Faker()
+
+
+class UserFactory(factory.django.DjangoModelFactory):
+    """Create a fake user."""
+    class Meta:
+        model = User
+
+    username = factory.Sequence(lambda n: 'user{}'.format(n))
+    first_name = fake.first_name()
+    last_name = fake.last_name()
+    email = fake.email()
+
+
+class PhotoFactory(factory.django.DjangoModelFactory):
+    """Create a fake photo."""
+    class Meta:
+        model = Photo
+
+    photo = factory.django.ImageField()
+    title = fake.sentence()
+    description = fake.text()
+    user = factory.SubFactory(UserFactory)
```

Comment:
good.  Also look into the `django_get_or_create` property of a factory.  It allows you to name a field or set of fields the values of which will be used to determine whether to create a new object, or use an existing one.